app/controllers/concerns/set_for_end_of_month.rb

```diff
@@ -3,7 +3,17 @@ def set_for_end_of_month
     # Defaults to current month if it's the last week of the month.
     @show_current_month = ((Date.current.end_of_month - Date.current).to_i <= 7)
 
-    @for_end_of_month ||= if params[:for_end_of_month]
+    Sentry.configure_scope do |scope|
+      user = @current_admin || @current_user
```

Comment:
Is it easy for us to pull this out and apply it to every controller action under `AdminController`?

Reply:
Not easy enough where I want to do it in this PR 😁 ...this module gets mixed into Admin and API controllers, so I want to keep this change isolated.

Reply:
Fair enough 😅